https://issues.redhat.com/browse/ACM-6946 Observability doc updates (#5444)
open-cluster-management/rhacm-docs: observability/observability_arch.adoc

[#observability-arch]
= Observability architecture

> The observability service can be used to get a status and metrics for your hub cluster and managed clusters. When you enable observability, the `multicluster-observability-operator` pod is deployed by the `multiclusterhub-operator` automatically to each imported or created hub cluster.

Reviewer: In this case, since we're already calling out the user specifically (your hub cluster), I'd make the entire sentence user focused/action oriented. 

Use the observability service to get a status and metrics for your hub cluster and managed clusters.

Author: Great point!

> The observability service can be used to get a status and metrics for your hub cluster and managed clusters. When you enable observability, the `multicluster-observability-operator` pod is deployed by the `multiclusterhub-operator` automatically to each imported or created hub cluster.

Author: incorporating suggestion after peer review
```suggestion
Use the observability service to get a status and metrics for your hub cluster and managed clusters. When you enable observability, the `multicluster-observability-operator` pod is deployed by the `multiclusterhub-operator` automatically to each imported or created hub cluster.
```

//didn't see the endpoint-operator in the diagram, is this pod still relevant? ---> was addressed during call and will double check (not a pod, thinks this is a container

//is the endpoint operator the same as the observaility add-on controller? 
//When the service is enabled, the `observability-endpoint-operator` is automatically deployed to each imported or created cluster. This controller collects the data from {ocp} Prometheus, then sends it to the {product-title-short} hub cluster. If the hub cluster imports itself as the `local-cluster`, observability is also enabled on it and metrics are collected from the hub cluster.

> //original

Reviewer: I assume the content in these three paragraphs are the hidden comments, hence, not needing review because this content is not being published?

Author: Correct

//The observability component deploys an instance of Prometheus Alertmanager, which enables alerts to be forwarded with third-party applications. It also includes an instance of Grafana to enable data visualization with dashboards (static) or data exploration. {product-title-short} supports version 8.5.20 of Grafana. You can also design your Grafana dashboard. For more information, see _Designing your Grafana dashboard_. You can customize the observability service by creating custom recording rules or alerting rules.
 
//need to update the diagram for 2.8 and 2.9, 
//D suggested to add a box for Thanos on the hub cluster side + metrics-collector on the managed cluster side, get managed cluster metrics are collected and sent to the hub cluster | MJ | 08/30/23

> The following diagram shows the components of the observability component:

Reviewer: Minimalism: avoid redundancy. 

The following diagram shows the observability components:

> The following diagram shows the components of the observability component:

Author: ```suggestion
The following diagram shows the components of observability:
```

image:../images/observability-arch-24.png[Multicluster observability architecture]

> *Note:* The _on-demand log_ provides access for engineers to get logs for a given pod in real-time. Logs from the hub cluster are not aggregated. These logs can be accessed with the search service and other parts of the console.

Reviewer: We've been using second person to refer to users throughout. Now, we're saying engineers. For consistency, can we use second person here? So something like:

*Note:* The _on-deand_ provides you access to get logs for your pods in real time. Logs from the hub cluster are not aggregated. To access the hub cluster logs, use the search service and other parts of the console.

> *Note:* The _on-demand log_ provides access for engineers to get logs for a given pod in real-time. Logs from the hub cluster are not aggregated. These logs can be accessed with the search service and other parts of the console.

Author: Incorporating suggestion after peer review
```suggestion
*Note:* The _on-demand log_ provides you access to get logs for your pods in real time. Logs from the hub cluster are not aggregated. To access the hub cluster logs, use the search service and other parts of the console.
```

The components of the observability architecture include the following items:

> - The multicluster hub operator (`multiclusterhub-operator` pod) deploys the `multicluster-observability-operator` pod. It sends hub cluster data to your managed clusters.

Reviewer: IBM SG wants us to avoid parenthesis unless it's specifying an acronym. I think we can simply say:

The multicluster hub operator, or the `multicluterhub-operator` pod, deploys the `multicluster-observability-operator` pod.

> - The multicluster hub operator (`multiclusterhub-operator` pod) deploys the `multicluster-observability-operator` pod. It sends hub cluster data to your managed clusters.

Author: Incorporating suggestion after peer review
```suggestion
- The multicluster hub operator, also known as the `multiclusterhub-operator` pod, deploys the `multicluster-observability-operator` pod. It sends hub cluster data to your managed clusters.
```
